source/crud/bulk.txt

```diff
@@ -623,6 +623,54 @@ Even though the write operation inserting a document with a duplicate key result
 in an error, the other operations are executed because the write operation is
 unordered.
 
+.. _java-bulk-insert-troubleshooting:
+
+Troubleshooting
+---------------
+
+Bulk Write Exception
+~~~~~~~~~~~~~~~~~~~~
+
+If the driver encounters an error during a bulk write operation, the driver
+throws a `MongoBulkWriteException.
+<{+core-api+}/MongoBulkWriteException.html>`__ A ``MongoBulkWriteException``
+contains a ``writeErrors`` field consisting of a list of one or more
+``WriteError`` objects associated with the same bulk write operation. 
+
+Consider a collection that has a rule where the value of the ``quantity`` field
```

Comment:
I think it's useful to emphasise that we're talking about "schema validation rule"